src/common/Components/HeaderWrapper/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Header } from '../../../app/Components';
+
+const propTypes = {
+  children: PropTypes.object.isRequired,
+  position: PropTypes.string,
+};
+
+const defaultProps = {
+  position: '',
+};
+
+const HeaderWrapper = ({ children, position }) => (
+  <>
+    <Header position={position} />
+    {children}
+  </>
+);
```

Comment:
We can discuss this approach later, but what's the benefit of having this wrapper and specifying it on each route?